src/fortran_routing/msh_fortran_CompCh/input/bed_elevation.txt

```diff
@@ -0,0 +1,501 @@
+0.0 5.0
```

Comment:
This doesnt seem like a .txt file, better to call it something else since it contains very little text

Reply:
I think they can stay as .txt for now -- eventually, we need to be pulling input files from something resembling output from the national water model, so we can wait for that to make a systematic change, I think.